pkgs/development/compilers/cudatoolkit/default.nix

```diff
@@ -90,5 +90,5 @@ rec {
   # latest cudnn, nccl, cutensor, etc! It sometimes happens that CUDA versions
   # are released prior to compatibility with the rest of the ecosystem. And
   # don't forget to request a review from @NixOS/cuda-maintainers!
-  cudatoolkit_11 = cudatoolkit_11_5;
+  cudatoolkit_11 = cudatoolkit_11_4; # update to 11.5 or 11.6 when pytorch reaches 1.11
```

Comment:
This is from https://github.com/NixOS/nixpkgs/pull/167250#issuecomment-1088139725
Sorry for confusion